### PR TITLE
Analysis of mean power ratio in MEMA

### LINKED ISSUE #51: Add ratio of power in the reporting for middle ear events
It could be cool to report the mean ratio of power between events and (adaptive) baseline values:
in fact, one hypothesis to test is "are MEMA activity for tinnitus patients more powerful than for controls". When I inspect data of "pure" middle ear events, I think it's possible this hypothesis is true.

### PR DESCRIPTION
**Pull request by lkorczowski**
- [x] mean power ratio
- [x] pytest
- [x] check if there is at least one in numerator and denominator
- [x] add to reporting

closes #51

### CODE REVIEW

**Comment by lkorczowski** [comment]
@RobinGuillard before doing power ratio analysis, we should get 250 Hz from #61 

Standing by

**Comment by RobinGuillard** [comment]
Everything ready for merge on #61, please apply solution of #63 and proceed to merge

File: tinnsleep/signal.py [hunk 1]
```diff
@@ -23,7 +23,50 @@ def rms(epochs, axis=-1):
...
+    epochs : ndarray, shape (n_trials, n_electrodes, n_samples)
+        the epochs for the estimation
+    axis : None or int or tuple of ints, optional (default: -1)
+        Axis or axes along which the means are computed.
+    labels : ndarray | list, shape (n_trials,)
+        A list of boolean. True are for numerator, False for the denominator
+
+    Returns
+    -------
+    mean_power : ndarray, shape (n_trials, n_electrodes)
```
Comment (RobinGuillard) [inline tinnsleep/signal.py:60]: Sure that the output is not rather (n_electrodes,) this would be more coherent with your pytests...

Comment (lkorczowski) [reply tinnsleep/signal.py]: done

**Pull request merged**